### ISSUE TITLE
Unable to install applet on card

### ISSUE THREAD

**Issue by mgramigna** [arekinath/PivApplet#4]
Hi, I am following [these directions](https://github.com/OpenSC/OpenSC/wiki/Smart-Card-Simulation) to simulate PIV smart card login on Windows. I am able to have the virtual reader recognize the PIV card.

When I run `gp -install PivApplet-jc304.cap`, I get the following error:

`Could not auto-detect ISD AID: 613A4F0BA000000308000010000100790D4F0BA00000030800001000010050095069764170706C6574AC118001038001068001078001118001F00600`

If relevant, this is the following output of `gp -info`:

"GlobalPlatformPro 18.08.16.1-0-g517663b
Running on Windows 7 6.1 x86, Java 1.8.0_181 by Oracle Corporation
Reader: Fabio Ottavi Pipe Reader 0
ATR: 3B80800101
More information about your card:
    http://smartcard-atr.appspot.com/parse?ATR=3B80800101"

Let me know if I can provide any more information on the issue. Thanks!

**Comment by martinpaljak** [comment]
Correct, gp is trying to communicate with an already emulated piv applet in this case. There is no installation step with such emulation

**Comment by arekinath** [comment]
Personally, I've mostly tested on NXP cards -- the J3D081 is a good choice since it'll run all the EC bits in this applet well, but the more common J2A040 and family will do all the RSA ops just fine. I've also used some cards from Feitian, including rebranded ones you can find on Amazon.
